content/_changelogs/11.0.0.md

```diff
@@ -1,17 +1,105 @@
 ## 11.0.0
 
-_Released MM/DD/YYYY_
+_Released 11/08/2022_
 
 **Summary:**
 
+Component Testing is now generally available for projects using React, Next.js,
+Angular, and Vue!
+
+Component tests allow you to see and test your application's components in a
+real browser as you work. You can use your favorite Cypress commands and
+features to develop your components without running your whole app. Learn more
+in our [blog](https://on.cypress.io/cypress-11-release) post.
+
+We have also massively improved our startup performance by shipping a snapshot
+of our binary instead of the source files. Results will vary based on your
+situation, but we saw up to 84% faster startup times!
+
 **Breaking Changes:**
 
-**Deprecations:**
+**Read our
```

Comment:
Added [7ae5a18](https://github.com/cypress-io/cypress-documentation/pull/4844/commits/7ae5a185f240140ad11181eb03b47dcc14e15df5)

Reply:
Since we now have breaking changes not related to component testing, I decided to make three sections under breaking, `Component Testing`, `General`, and `Experimental`. [8526a30](https://github.com/cypress-io/cypress-documentation/pull/4844/commits/8526a30a7a76872a43bd1d18a66315b229e0b126)